### ISSUE TITLE
“alphaGen lightingSpecular” shader keyword is unsupported

### ISSUE THREAD

**Issue by illwieckz** [DaemonEngine/Daemon#213]
Many Tremulous maps use it, but Dæmon does not implement it:

```
Warn: alphaGen lightingSpecular keyword not supported in shader 'textures/metro/zinc_ed' 
```

May be good to have this shader supported or at least to have a fallback to make map porting easier.

**Comment by megatog615** [comment]
Back in the day, it was intended to be used with texture maps that were made to have the surface look shiny with specular lighting. It's an alphaGen mode because the texture map made for it is intended to blend through the alpha channel. The alpha channel increases in value based on a predetermined direction(q3a had built-in map coordinates for this rather than using any lights). So, it's a bit like modern specular mapping but one thing to consider is the texture map made for alphaGen lightingSpecular is *not* your typical specular/gloss map and you'll have to reimplement the weird use-case introduced by blending it through the alpha channel.

**Comment by VReaperV** [comment]
It's better to cover this alpha-channel specular map into a proper specular map instead.

**Comment by illwieckz** [comment]
Yep but I want it to be supported for legacy maps.

**Comment by VReaperV** [comment]
Yeah, I'm just saying we don't need some hacky extra code path in glsl shaders for it.
